packages/astro-ghostcms/virtual-config.d.ts

```diff
@@ -1,4 +1,4 @@
 declare module "virtual:@matthiesenxyz/astro-ghostcms/config" {
-	const Config: import("../schemas/index").GhostUserConfig;
+	const Config: import("./src/schemas/userconfig").GhostUserConfig;
```

Comment:
Is this virtual file one that's copied to the user's project? Or is this an internal dev one? If it gets copied with like `injectDts` just make sure this relative path is ok

Reply:
internal dev.  the virtual for config is not used for anything aside from grabbing the ghostURL for the API :P